src/components/BaseMap.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+	<div id="map" :style="mapHeight">
+		<gmap-map :zoom="14" :center="center" :style="mapHeight">
+			<gmap-marker
+				:key="index"
+				v-for="(m, index) in locationMarkers"
+				:position="m.position"
+				@click="center = m.position"
+			></gmap-marker>
+		</gmap-map>
+	</div>
+</template>
+<script>
+export default {
+	props: {
+		height: {
+			type: String,
+			default: "475px",
+		},
+		lat: {
+			type: Number,
+			default: -25.344,
+		},
+		lng: {
+			type: Number,
+			default: 131.036,
+		},
+	},
+	data() {
+		return {
+			center: {
+				lat: this.lat,
+				lng: this.lng,
+			},
+			locationMarkers: [],
+			locPlaces: [],
+			existingPlace: null,
+		}
+	},
+	computed: {
+		mapHeight() {
+			return `height: ${this.height}`
+		},
```

Comment:
I guess it's better to use number in height prop and just return formatted value here 
`return height: ${this.height}px`

Reply:
I had used String because I wanted to have the opportunity to use `%` or `vh`. Not just `px`